openshift/keycloak.app.yaml

```diff
@@ -0,0 +1,135 @@
+kind: Template
+apiVersion: v1
+metadata:
+  creationTimestamp: null
+  name: keycloak
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    creationTimestamp: null
+    generation: 1
+    name: keycloak-server
+  spec:
+    strategy:
+      type: Rolling
+    triggers:
+    - type: ConfigChange
+    test: false
+    replicas: 1
+    selector:
+      name: keycloak-server
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          name: keycloak-server
+      spec:
+        containers:
+        - name: keycloak-server
+          image: registry.devshift.net/almighty/keycloak-postgres
+          imagePullPolicy: Always
+          ports:
+          - containerPort: 8080
+            protocol: TCP
+          resources: {}
+          livenessProbe:
+            httpGet:
+              path: /auth/version
+              port: 8080
+              scheme: HTTP
+            successThreshold: 1
+            failureThreshold: 3
+            timeoutSeconds: 1
+            periodSeconds: 10
+            initialDelaySeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /auth/realms/fabric8
```

Comment:
Yes, let's use fabric8 for now. It won't work for an empty DB until we add proper configuration exportation though. But it's safer for now to relay on fabric8.